pkg/cmd/pr/merge/merge.go

```diff
@@ -333,6 +333,12 @@ func mergeRun(opts *MergeOptions) error {
 			if err != nil {
 				return err
 			}
+
+			err = git.PullLatestChanges()
+			var httpErr api.HTTPError
```

Comment:
Git commands can never result with an `api.HTTPError`. Perhaps you wanted `*os.ExitError`